### PR TITLE
Disable tests using extra binaries

### PR DESCRIPTION
**Pull request by blender**
@jgm  Should solve #41 while keeping tests active on CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -129,10 +129,10 @@ script:
       stack --no-terminal $ARGS test --fast --flag 'aeson:fast' --flag 'texmath:executable' --flag 'zip-archive:enable_extra_tools' --haddock --no-haddock-deps --ghc-options="$OPTS"
       ;;
     cabal)
-      cabal configure --enable-tests --enable-benchmarks -v2 -ffast -fexecutable --ghc-options="$OPTS" && \
+      cabal configure --enable-tests --enable-benchmarks -v2 -ffast -fexecutable -fenable_extra_tools --ghc-options="$OPTS" && \
       cabal sdist --output-directory=sourcedist && \
       cd sourcedist && \
-      cabal configure --enable-tests --enable-benchmarks -v2 -ffast -fexecutable --ghc-options="$OPTS" && \
+      cabal configure --enable-tests --enable-benchmarks -v2 -ffast -fexecutable -fenable_extra_tools --ghc-options="$OPTS" && \
```
Comment (fisx) [inline .travis.yml]: why configure twice?  (i'm assuming this has nothing to do with this PR, just curious.)

Comment (blender) [reply .travis.yml]: No idea :\

Comment (fisx) [reply .travis.yml]: oh well, as long as it works?  (-:

Comment (jgm) [reply .travis.yml]: You need to configure before generating an sdist.
Then, after switching to the sdist, we need to configure again in that directory.

**Comment by blender** [comment]
Closing since problem was resolved with custom stanza

**Pull request closed without merging**